src/mscorlib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -408,7 +408,7 @@ public static void Copy(IntPtr source, IntPtr[] destination, int startIndex, int
         //====================================================================
         public static byte ReadByte([MarshalAs(UnmanagedType.AsAny), In] Object ptr, int ofs)
         {
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(); // https://github.com/dotnet/coreclr/issues/10442
```

Comment:
I didn't include a message because these are in the category of "maybe we should implement" (hence the issue link also). THere's an email thread about it.